impeller/entity/entity.cc

```diff
@@ -71,7 +71,14 @@ Contents::ClipCoverage Entity::GetClipCoverage(
 }
 
 bool Entity::ShouldRender(const std::optional<Rect>& clip_coverage) const {
-  return contents_->ShouldRender(*this, clip_coverage);
+  if (!clip_coverage.has_value()) {
+    return false;
+  }
+#ifdef IMPELLER_CONTENT_CULLING
```

Comment:
Do we have a planned end-of-line for this flag?  If we decide we don't want ShouldRender, we should just rip it out.  It's going to atrophy otherwise.

Reply:
We're going to turn this on in impeller-cmake for now (which isn't yet located in the engine repo) since this is the only frustum culling mechanism in Impeller.